pages/library/_viewId.vue

```diff
@@ -54,6 +52,20 @@ export default Vue.extend({
 
       this.items = itemsResponse.data.Items || [];
     }
+  },
+  methods: {
+    imageLink(id: string) {
+      return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Primary`;
+    }
   }
 });
 </script>
+
+<style scoped>
+.cardsContainer {
```

Comment:
This can use the Vuetify flex helpers instead: https://vuetifyjs.com/en/styles/flex/